### DISCUSSION TITLE
platform.uno crash

### DISCUSSION THREAD

**Discussion by limeniye** [unoplatform/uno#11109] [General]
I'm trying to expand some menu point. But I have next result:

![thesitedropdown](https://user-images.githubusercontent.com/69314237/215617309-6584cab6-5330-4a52-a3ae-fdba4bbbb854.gif)

The link:
https://platform.uno/docs/articles/external/uno.extensions/doc/Learn/Tutorials/Navigation/HowTo-NavigateInCode.html

Browsers:
- Google
- Mozilla

**Comment by Xiaoy312** [comment]
Thanks for the report.

We have identified the cause of the issue to be with the js minifier.
A fix for this on the way: #11304
~~Will have to wait for it to come live now. (I don't know the timeline for this.)~~
edit20230222: It is now live as of written.
